dev-java/jetty-util/jetty-util-9.3.5.20151012.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+
+JAVA_PKG_USE="doc source"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="Jetty's Utils"
+
+MY_PN="jetty"
+MY_PV="${PV/2015/v2015}"
+MY_P="${MY_PN}-${MY_PV}"
+
+SLOT="$(get_version_component_range 1-2)"
```

Comment:
You go in two opposing directions at the same time, so pick one direction or the other.

Either all java ebuilds need to inherit java-utils-2, or we can rely on the fact that java-utils-2 inherits versionator and that will never change. Pick one, you cannot have both as they oppose each other.

You may want to review versionators use in java-utils-2 and determine if that is something you feel is optional and can be removed, replaced with something else. Which at best would be like versionator-2, I doubt one would re-code function found in versionator directly in java-utils-2.

With that being the case, every java ebuild needs to be updated to inherit java-utils-2. For the exact reason they need to inherit versionator. Your argument cannot be subjective it must be consistent. Either do it one way for all, or not. Which will be it?